src/pages/my-cooperations/MyCooperations.tsx

```diff
@@ -86,16 +89,23 @@ const MyCooperations = () => {
     value
   }))
 
+  const { userRole } = useAppSelector((state) => state.appMain)
+
+  const buttonPath =
+    userRole === 'student'
```

Comment:
Use studentOrTutor function and pass RoleEnum.Student

Reply:
fixed